package.json

```diff
@@ -13,6 +13,7 @@
         "@heroicons/react": "^2.2.0",
         "firebase": "^11.0.2",
         "react": "^18.3.1",
+        "react-calendar": "^5.1.0",
```

Comment:
Brianna knows something about the calendly import - it's possible it may be better than react-calendar.